code/modules/mob/living/living.dm

```diff
@@ -1031,6 +1031,9 @@
 	if(G.trigger_guard != TRIGGER_GUARD_ALLOW_ALL && !IsAdvancedToolUser() && !issmall(src))
 		to_chat(src, "<span class='warning'>You don't have the dexterity to do this!</span>")
 		return 0
+	if(G.trigger_guard == TRIGGER_GUARD_NONE)
+		to_chat(src, "<span class='warning'>This gun is only built to be fired by machines!</span>")
+		return 0
```

Comment:
use false rather than 0

Reply:
return false instead of return 0? 0 was what the original code used